Dockerfile

```diff
@@ -1,20 +1,38 @@
-FROM debian:latest
+FROM debian:stretch
 MAINTAINER Ondrej Vasko
 
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt-get update 
-RUN apt-get install -y locales && rm -rf /var/lib/apt/lists/* && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
-ENV LANG en_US.utf8
+RUN apt-get update && apt-get install -y \
+  locales \
+  postfix \
+  syslog-ng \
+  syslog-ng-core 
 
-RUN apt-get update 
-RUN apt-get install -y postfix supervisor syslog-ng syslog-ng-core
+RUN rm -rf /var/lib/apt/lists/* 
```

Comment:
i'd move to this to upper layer (apt-get update & install ...)

Reply:
Like above of apt-get?